spec/models/manageiq/providers/azure/cloud_manager/provision/cloning_spec.rb

```diff
@@ -113,4 +116,33 @@
       expect(provision.create_nic(false)).to eql(@nic_object.id)
     end
   end
+
+  context "start_clone_task" do
+    before do
+      @resource_group = FactoryBot.create(:azure_resource_group)
+
+      
```

Comment:
blank lines? (accidental commit?)

Reply:
I asked him to push up whatever he had so I could help him with his failing spec